index.html

```diff
@@ -105,7 +105,12 @@ <h2>Multi-Post Stories Gain+Glory</h2>
                     <li class="languagesdef">html</li>
                   </ul>
                   <div class="buttoncont">
-                    <a class="button" href="#">See Project</a>
+                    <a
+                      class="button"
+                      id="project1"
+                      onclick="CreatePop(projects[0])"
+                      >See Project</a
+                    >
```

Comment:
- [ ] Kindly make sure the popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, tests, margins).👍
- Figma mobile popup
![screenshot (22)](https://user-images.githubusercontent.com/44624138/147363879-2f8126a6-ee18-47e9-8c14-533d37376a15.png)
- Your mobile popup
![screenshot (23)](https://user-images.githubusercontent.com/44624138/147363910-615861c2-f43c-4b78-b45a-3f0844fbcc2f.png)